code/game/machinery/merch_vendor.dm

```diff
@@ -74,10 +77,13 @@
 	user.put_in_hands(D)
 	SSeconomy.total_vendor_transactions++
 
-/obj/machinery/economy/merch/ui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = TRUE, datum/tgui/master_ui = null, datum/ui_state/state = GLOB.physical_state)
-	ui = SStgui.try_update_ui(user, src, ui_key, ui, force_open)
+/obj/machinery/economy/merch/ui_state(mob/user)
+	return GLOB.physical_state
```

Comment:
any particular reason why the merch computer uses a non-default state?

Reply:
Not beyond "this is how it always was" 
![image](https://github.com/ParadiseSS13/Paradise/assets/12197162/49c6f7b3-1b8c-4929-9df0-fed0b6917aa6)

Reply:
Have changed it to default